packages/jazz-browser-auth-clerk/src/index.ts

```diff
@@ -1,26 +1,9 @@
+import type { Clerk } from "@clerk/clerk-js";
```

Comment:
If we depend on the type this is not a devDependency but an actual dependency.

I guess the `MinimalClerkClient` type was inlined with the intention of avoiding that.

Could you revert the change and update the local type?